modules/ocf_desktop/manifests/drivers.pp

```diff
@@ -1,4 +1,8 @@
 class ocf_desktop::drivers {
+  class { 'ocf::apt::i386':
```

Comment:
yeah, i think that makes sense. i moved it from steam to drivers because it seemed better, but init is probably better than both

puppet sucks sometimes